include/uv/unix.h

```diff
@@ -124,12 +124,18 @@ typedef int uv_os_sock_t;
 typedef int uv_os_fd_t;
 typedef pid_t uv_pid_t;
 
-#define UV_ONCE_INIT PTHREAD_ONCE_INIT
+typedef struct UV_CAPABILITY("uv_once") uv_once_s {
+  pthread_once_t o;
+} uv_once_t;
```

Comment:
That's technically a backwards incompatible change. Compiling against musl (and, I think, glibc too), you can do this currently:
```c
uv_once_t a, b;
a = b = UV_ONCE_INIT;
uv_once(&a, init1);
if (1 & rand()) uv_once(&b, init2);
if (a == b) puts("both initialized");
```
Contrived for expository reasons; the point is that `uv_once_t` is a simple value that you can copy and compare.

Hard to gauge how much fallout such a change would cause in practice but I'd rather err on the side of caution.